llvm/test/CodeGen/AArch64/sme-streaming-body-streaming-compatible.ll

```diff
@@ -0,0 +1,177 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc -mtriple=aarch64-linux-gnu -mattr=+sme -start-after=simplifycfg -enable-tail-merge=false -verify-machineinstrs < %s | FileCheck %s
+
```

Comment:
Could you also add a test where with multiple basic blocks?

Reply:
Done.